src/app/venter/complaint-card/complaint-card.component.html

```diff
@@ -0,0 +1,73 @@
+<div class="complaint-card" *ngFor="let complaint of complaintList; let i = index">
```

Comment:
1) If you're making a component for a card, then it should be a *single* card, not a list of them. Put this loop where this component is being used.
2) Might be missing something, but I don't see `i` being used anywhere.